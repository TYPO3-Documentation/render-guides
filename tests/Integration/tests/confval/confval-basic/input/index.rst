=======
Confval
=======

..  confval:: demo
    :type: string
    :default: ``"Hello World"``
    :required: False

    Some Text

See also :confval:`demo`.
