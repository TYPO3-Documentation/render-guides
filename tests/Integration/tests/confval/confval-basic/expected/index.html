<!-- content start -->
                <section class="section" id="confval">
            <h1>Confval<a class="headerlink" href="#confval" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">¶</a></h1>
            <dl class="confval">
    <dt id="demo">
        <code class="sig-name descname"><span class="pre">demo</span></code>
        <a class="headerlink" href="#demo" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  data-id="demo" data-rstCode=":confval:`demo &lt;somemanual:demo&gt;`"  title="Reference this configuration value">¶</a>
    </dt>
    <dd>
        <div class="line-block">
             <div class="line"><strong>Type:</strong>  <code>&quot;Hello World&quot;</code></div>
             <div class="line"><strong>Required:</strong> true</div>
            </div>
        <div class="confval-description">

    <p>Some Text</p>

        </div>
    </dd>
</dl>

    <p>See also <a href="#demo">demo</a>.</p>

    </section>
        <!-- content end -->
